From 50e2577b02cd5d3558165daefaf54b27c6dff479 Mon Sep 17 00:00:00 2001
From: Vladimir Barinov <vladimir.barinov@cogentembedded.com>
Date: Wed, 4 Jan 2017 12:49:30 +0300
Subject: [PATCH] arm64: dts: r8a7795 view boards: stream from 7 cam

csi2 stream from 7 cameras on view boards:
VIN[0-3] and VIN[4-6]

Signed-off-by: Vladimir Barinov <vladimir.barinov@cogentembedded.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-h3ulcb-had.dts      | 4 ++--
 arch/arm64/boot/dts/renesas/r8a7795-h3ulcb-kf.dts       | 4 ++--
 arch/arm64/boot/dts/renesas/r8a7795-h3ulcb-view.dts     | 4 ++--
 arch/arm64/boot/dts/renesas/r8a7795-salvator-x-view.dts | 4 ++--
 4 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb-had.dts b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb-had.dts
index c88edc6..ddef8df 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb-had.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb-had.dts
@@ -689,7 +689,7 @@
 		reg = <0x6c>;
 		gpios = <&gpio5 25 GPIO_ACTIVE_HIGH>;
 		maxim,sensor_delay = <0>;
-		maxim,links = <4>;
+		maxim,links = <3>;
 		maxim,resetb-gpio = <1>;
 		maxim,resetb-active-low;
 		maxim,fsync-mode = "automatic";
@@ -1135,7 +1135,7 @@
 	status = "okay";
 
 	adi,input-interface = "ycbcr422";
-	csi2,csi_rate = <700>;
+	csi2,csi_rate = <550>;
 
 	virtual,channel {
 		csi2_vc0 {
diff --git a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb-kf.dts b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb-kf.dts
index 0914af9..8530b10 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb-kf.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb-kf.dts
@@ -1136,7 +1136,7 @@
 				compatible = "maxim,max9286-max9271";
 				reg = <0x2a>;
 				maxim,sensor_delay = <250>;
-				maxim,links = <4>;
+				maxim,links = <3>;
 				maxim,resetb-gpio = <1>;
 				maxim,resetb-active-low;
 				maxim,fsync-mode = "automatic";
@@ -1800,7 +1800,7 @@
 	status = "okay";
 
 	adi,input-interface = "ycbcr422";
-	csi2,csi_rate = <700>;
+	csi2,csi_rate = <550>;
 
 	virtual,channel {
 		csi2_vc0 {
diff --git a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb-view.dts b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb-view.dts
index 8265b56..683dd18 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb-view.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb-view.dts
@@ -574,7 +574,7 @@
 		reg = <0x6c>;
 		gpios = <&gpio5 25 GPIO_ACTIVE_LOW>;
 		maxim,sensor_delay = <0>;
-		maxim,links = <4>;
+		maxim,links = <3>;
 		maxim,resetb-gpio = <1>;
 		maxim,resetb-active-low;
 		maxim,fsync-mode = "automatic";
@@ -1017,7 +1017,7 @@
 	status = "okay";
 
 	adi,input-interface = "ycbcr422";
-	csi2,csi_rate = <700>;
+	csi2,csi_rate = <550>;
 
 	virtual,channel {
 		csi2_vc0 {
diff --git a/arch/arm64/boot/dts/renesas/r8a7795-salvator-x-view.dts b/arch/arm64/boot/dts/renesas/r8a7795-salvator-x-view.dts
index fa52ba8..adf6f30 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-salvator-x-view.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-salvator-x-view.dts
@@ -866,7 +866,7 @@
 		compatible = "maxim,max9286-max9271";
 		reg = <0x6c>;
 		maxim,sensor_delay = <0>;
-		maxim,links = <4>;
+		maxim,links = <3>;
 		maxim,resetb-gpio = <1>;
 		maxim,resetb-active-low;
 		maxim,fsync-mode = "automatic";
@@ -1430,7 +1430,7 @@
 	status = "okay";
 
 	adi,input-interface = "ycbcr422";
-	csi2,csi_rate = <700>;
+	csi2,csi_rate = <550>;
 
 	virtual,channel {
 		csi2_vc0 {
-- 
1.9.1

